File: Ruby/fizzbuzz-inline/fizzbuzz-inline.rb
# Simple example of FizzBuzz inline with ruby 
# Author: @Tony133

1.upto(100) { |n| puts "#{'Fizz' if n % 3 == 0}#{'Buzz' if n % 5 == 0}#{n if n % 3 != 0 && n % 5 != 0}" }

File: javascript/fizzbuzz-inline/fizzbuzz-inline.js
// Simple example of FizzBuzz inline with javascript 
// Author: @Tony133

let i, values = [, , 'fizz', , 'buzz', 'fizz', , , 'fizz', 'buzz', , 'fizz', , , 'fizzbuzz'];

for (i = 0; i < 100; console.log(values[i++ % 15] || i));
